resources/qml/Toolbar.qml

```diff
@@ -53,7 +53,7 @@ Item
                 width: childrenRect.width
                 height: childrenRect.height
 
-                delegate: ToolbarButton
+                delegate: UM.ToolbarButton
                 {
                     text: model.name + (model.shortcut ? (" (" + model.shortcut + ")") : "")
```

Comment:
not really part of this commit but I think the parenthesis here make the code a bit unclear. I think [javascripts template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) could really benefit here
```suggestion
                    text: model.name + (model.shortcut ? ` ("${model.shortcut}")` : "")
```